ganga/GangaGUI/test/test_jobs_api.py

```diff
@@ -0,0 +1,109 @@
+# ******************** Imports ******************** #
+
```

Comment:
Try to avoid self-evident comment lines.

Reply:
Okay, I will make the changes in the next PRs